packages/Dispatch/readme.md

```diff
@@ -8,6 +8,18 @@ Use the Dispatch model in cases when:
 2. Evaluate quality of intents classification of a single LUIS model.
 3. Create a text classification model from text files.
 
+## Deprecation Roadmap
+Dispatch command line is on path to be replaced with [Orchestrator](https://aka.ms/bf-orchestrator) recognizer. Orchestrator is an independent technology from LUIS and QnAMaker. 
```

Comment:
"Dispatch CLI is on the path to be replaced with [Orchestrator](https://aka.ms/bf-orchestrator) CLI. Orchestrator is an independent technology from LUIS and QnAMaker. "

Recognizer is run time terminology...